client/my-sites/feature-upsell/features.jsx

```diff
@@ -105,7 +105,9 @@ class FeaturesComponent extends Component {
 						description={ 'Attract new (and more!) traffic immediately with Google Ads.' }
 						body={
 							<div className="google-voucher__initial-step">
-								<TipInfo info={ 'Offer valid in US after spending the first $25 on Google Ads.' } />
+								<TipInfo
+									info={ 'Offer valid in US and CA after spending the first $25 on Google Ads.' }
```

Comment:
We should extract the price.

Reply:
Nothing from this file is translated and I think this is done on purpose.

Reply:
Sorry, I didn't realize that.